### PR TITLE
[IMP]add configuration menus for web, e-commerce, marketing, pos

### PR DESCRIPTION
**Pull request by msa-odoo**
Add new configuration menus for web, e-commerce, marketing, pos...

Task: https://www.odoo.com/web#id=10280&view_type=form&model=project.task&menu_id=3942&action=327

Pad:https://pad.odoo.com/p/r.w7a97byyL9mz5nln

### CODE REVIEW

File: addons/sale/res_config.py [hunk 1]
```diff
@@ -83,6 +83,22 @@ class sale_configuration(osv.TransientModel):
         'group_sale_delivery_address': fields.boolean("Allow a different address for delivery and invoicing ",
             implied_group='sale.group_delivery_invoice_address',
             help="Allows you to specify different delivery and invoice addresses on a sales order."),
+        'module_product_visible_discount': fields.boolean("Allow to you calculate discount on sale order line.",
+            help='This module lets you calculate discounts on Sale Order lines and Invoice lines base on the partners pricelist.\n'
+                 '-This installs the module product_visible_discount'),
+        'module_delivery': fields.boolean("Allows you to add delevery methods in sale orders and piking.",
```
Comment (atp-odoo) [inline addons/sale/res_config.py]: typo : sale orders and piking. ?

Comment (msa-odoo) [inline addons/sale/res_config.py]: Done.

File: addons/sale/res_config_view.xml [hunk 2]
```diff
@@ -70,6 +82,25 @@
                         </div>
                     </group>
                 </group>
+                <group name="Sale" position="after">
+                    <group name="E-Commerce Features">
+                        <label for="id" string="E-Commerce Features"/>
```
Comment (jem-odoo) [inline addons/sale/res_config_view.xml]: Why adding a label ? `string` attribute on `group` tag did not do the job ?
`name` attribute above is not made for that : it should contain an 'identifier' in underscore notation to apply xpath in an easy way.

File: addons/sale/res_config.py [hunk 3]
```diff
@@ -83,6 +83,22 @@ class sale_configuration(osv.TransientModel):
...
+        'module_delivery': fields.boolean("Allow you to add delivery methods in sale orders and picking",
+            help='Allows you to add delivery methods in sales orders and delivery orders.\n'
+                 'You can define your own carrier and delivery grids for prices.\n'
+                 '-This installs the module delivery.'),
+        'module_website_sale_options': fields.boolean("Allow to sale option your products online",
+            help='Allows you to display optional products on shop \n'
+                 '-This installs the module website_sale_option'),
+        'module_website_sale': fields.boolean("Allow to sale your product online",
+            help='Allows you to sale products online \n'
+                 'This installs the module website_sale'),
```
Comment (jem-odoo) [inline addons/sale/res_config.py]: For all your diff, add `-` before the sentence "This installs the module ..."

File: addons/website/views/res_config.xml [hunk 4]
```diff
@@ -86,6 +86,23 @@
                                 name="%(website.action_website_menu)d"
                                 string="Configure website menus" class="oe_link"/>
                         </group>
+                        <group string="Website">
+                            <label for="id" string="Website"/>
```
Comment (jem-odoo) [inline addons/website/views/res_config.xml]: same question as above

File: addons/website/views/res_config.xml [hunk 5]
```diff
@@ -86,6 +86,23 @@
                                 name="%(website.action_website_menu)d"
                                 string="Configure website menus" class="oe_link"/>
                         </group>
+                        <group string="Website">
```
Comment (jem-odoo) [inline addons/website/views/res_config.xml]: add a `name` attribute

File: addons/sale/res_config_view.xml (hunk 2 above)
Comment (atp-odoo) [inline addons/sale/res_config_view.xml]: Hello @jem-odoo  yes label is required here. string attribute with group tag didnot work.

**Pull request closed without merging**